_posts/2020-04-24-roboarmcontroller-intro.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title:  "RoboArm Controller, part I: Intro and architecture."
+date:   2020-04-28T09:25:52+01:00
+author: Enrique Llerena Dominguez
+categories: learning-machine-learning machine-learning 
+tags: deeplearning4j tensorflow java pyhton gazebo architecture roboarm-controller
+cover:  "/assets/posts/roboarmcontroller-intro/roboarmcontroller-cover.jpg"
+cover_credits: Photo by David Fanuel on Unsplash
+---
+
+# Learning Machine Learning Series
+
+This is the first post of a series where I want to describe my journey learning machine learning, a topic I am 
+passionate about.
+
+# RoboArm Controller, part I: Intro and architecture
+
+# TL;DR
+
+- The RoboArm Controller enables a user to control a simulated robotic arm with the position of their hands.
+- Check out the code at the  [Github repo: ellerenad/RoboArmController][roboarmcontroller_repo] 
+- [Demo][roboarmcontroller_demo]
+- The following technologies were used: Java, Python, C++, Tensorflow, Deeplearning4j, Spring, JUnit, Gazebo, plus other tools.
+- The RoboArm Controller has 3 main components: a simulated robotic arm, a data transformation application, and the hands' sensor.
+- The machine learning part is on the data transformation: classify the position of the hands to get a movement instruction.
+- The RoboArm Controller covers the machine learning cycle: Gather the data, prepare the data, choose a model, train the model,
+ tune the hyperparameters, test the model, export the model, import the model into a "production" system, and use it to make predictions.
+- The project was presented at the [Java Forum Stuttgart 2019][JFS_roboarmcontroller] and the EclipseCon Europe 2019 ([video][eclipsecon_roboarmcontroller_video]/[slides][eclipsecon_roboarmcontroller_slides])
+- The RoboArm Controller is a hobby project I developed to practice different technologies in a ludic way.
+
+
+## Introduction
+
+The RoboArm Controller allows controlling a simulated robotic arm with the position of the hands of a person.
```

Comment:
The intro is a bit quick, the problem description starts too soon for my feeling. Spend 1-2 more sentenes describing the setup, e.g. that you have 3 servos on the arm that you want to control, etc. That makes understanding the following sentences easier (for those who don't have the project context)